src/parser/core/modules/ActionWindow/windows/BuffWindow.ts

```diff
@@ -47,7 +47,7 @@ export abstract class BuffWindow extends ActionWindow {
 		// or on an enemy actor (trick attack)
 		// enemies are not on the same team as the parser actor
 		const enemyTargets = this.parser.pull.actors
-			.filter(actor => actor.team !== this.parser.actor.team)
+			.filter(actor => actor.team > this.parser.actor.team) // Ignore actors with an 'unknown' team (team 0, where player's are team 1: Friend)
```

Comment:
i know it's not as much of a one-line-er, but i'm not a huge fan of treating enums as numbers - would be more self-descriptive to just check `&&!=unknown`

Reply:
I can live with that. Done!